src/mpid/ch4/generic/am/mpidig_am_msg.h

```diff
@@ -106,7 +106,8 @@ MPL_STATIC_INLINE_PREFIX void MPIDIG_recv_copy(void *in_data, MPIR_Request * rre
                             MPIDIG_REQUEST(rreq, count),
                             MPIDIG_REQUEST(rreq, datatype), 0, &actual_unpack_bytes);
         if (!rreq->status.MPI_ERROR && in_data_sz > actual_unpack_bytes) {
-            /* If the receive buffer had enough space, but we still
+            /* Truncation error has been checked at MPIDIG_recv_type_init.
+             * If the receive buffer had enough space, but we still
```

Comment:
This comment belongs to the previous patch, which this actual change was made.

Reply:
Will fix ... done.